cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/extensions/CIMCharacteristics.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.extensions;
+
+import com.powsybl.cgmes.conversion.elements.CgmesTopologyKind;
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.Network;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public interface CIMCharacteristics extends Extension<Network> {
```

Comment:
I think that you should call your extension it ````CimCharacteristics````. For CGMES, we in general choose ````Cgmes```` instead of ````CGMES````. What do you think ?

Reply:
You're right, I will change it